src/renderer/actions/general.js

```diff
@@ -100,19 +100,20 @@ export function useRefreshAccountsOrderingEffect({
   onMount?: boolean,
   onUnmount?: boolean,
 }) {
-  const refreshAccountsOrdering = useRefreshAccountsOrdering();
+  const comparator = useSortAccountsComparator();
+  const dispatch = useDispatch();
 
   useEffect(() => {
     if (onMount) {
-      refreshAccountsOrdering();
+      dispatch(reorderAccounts(comparator));
     }
 
     return () => {
       if (onUnmount) {
-        refreshAccountsOrdering();
+        dispatch(reorderAccounts(comparator));
       }
     };
-  }, [onMount, onUnmount, refreshAccountsOrdering]);
+  }, [comparator, dispatch, onMount, onUnmount]);
```

Comment:
🤔  we are no longer using the `useRefreshAccountsOrdering` ?

Reply:
```js
  const comparator = useSortAccountsComparator();
```
  
comparator will often change references. for instance, when countervalues reload

```js
  }, [comparator, dispatch, onMount, onUnmount]);
```
  
it’s injected in the useEffect, making it re-run on each countervalues change

```js
  if (onMount) {
      refreshAccountsOrdering();
      dispatch(reorderAccounts(comparator));
    }
```
    
in the effect, if the useRefreshAccountsOrderingEffect was used with onMount, it actually no longer just run “onMount” but on each countervalues update
and we absolutely not want account to reorder “on the fly” because it create click jacking. this need to happen at precise lifecycle, like loading Accounts screen, etc..